setup.py

```diff
@@ -74,6 +74,11 @@
     ],
 }
 
+if platform.python_implementation == 'CPython':
+    deps['evm-extra'].append('eth-hash[pysha3]')
```

Comment:
Instead of switching in `setup.py` which doesn't work as expected in all cases, can you instead use this syntax: 

https://github.com/ethereum/eth-utils/blob/master/setup.py#L44-L45